### PR TITLE
Add "race-network-and-cache".

### PR DESCRIPTION
**Pull request by yoshisatoyanagisawa**
Race network and cache has been suggested to mitigate slow storage access.  Let me add that to the final-form.md.

### CODE REVIEW

**Comment by yoshisatoyanagisawa** [comment]
@quasi-mod @sisidovski Will you take a look?

File: final-form.md [hunk 1]
```diff
@@ -233,6 +238,40 @@ addEventListener('install', (event) => {
 });
 ```
 
+### race network and cache storage
```
Comment (sisidovski) [inline final-form.md:241]: I'm wondering what to be returned if either or both paths failed. For example, the network wins the race but returned network error, both responses are not OK responses for some reasons.

File: final-form.md [hunk 2]
```diff
@@ -233,6 +238,40 @@ addEventListener('install', (event) => {
...
+
+To deal with devices with slow storage access, web developer may want to race
+network and cache storage.  In this case, if the cache torage access is too slow
```
Comment (quasi-mod) [inline final-form.md:244]: NIT: cache torage -> storage

File: final-form.md (hunk 1 above)
Comment (quasi-mod) [reply final-form.md:241]: +1 on the failure cases.

Comment (yoshisatoyanagisawa) [reply final-form.md:241]: successful response will be returned if either failure case.  If both, the network error will be returned.
I added some explanations below.

File: final-form.md (hunk 2 above)
Comment (yoshisatoyanagisawa) [reply final-form.md]: done.

**Review by sisidovski — APPROVED** [review]

**Pull request merged**